suites/quincy/rbd/tier-3_rbd_persistent_write_back_cache.yaml

```diff
@@ -312,3 +312,28 @@ tests:
       module: test_rbd_persistent_write_back_cache.py
       name: PWL cache creation with exclusive lock
       polarion-id: CEPH-83574719
+
+  - test:
+      abort-on-fail: true
+      config:
+        level: client                        # PWL at client
+        cache_file_size: 1073741824          # 1 GB
+        rbd_persistent_cache_mode: ssd       # "ssd" or "rwl" on pmem device
+        client: node5
+        drive: /dev/nvme0n1
+        cleanup: true
+        # validate_exclusive_lock: true
+        rep-pool-only: True
+        rep_pool_config:
+          pool: pool2
```

Comment:
can we take `pool name `and ` image name` from utils default config?

Reply:
This is a standard we have followed for all persistent cache test cases till now, I would like to retain this for now. Maybe when we do SDK implementation for these we can consider these optimisations.